File: Examples/HelloWorld/HelloWorld.java
// SPDX-FileCopyrightText: 2026 Progmasoft <support@progmasoft.com>
// SPDX-License-Identifier: MPL-2.0 WITH AdditionRef-Progmasoft-Exception-1.0

public final class HelloWorld {
    private HelloWorld() {}

    public static void main(String[] args) {
        var language = "Java 21";
        System.out.printf("Hello from %s!%n", language);
    }
}

File: Examples/HelloWorld/HelloWorld.cpp
// SPDX-FileCopyrightText: 2026 Progmasoft <support@progmasoft.com>
// SPDX-License-Identifier: MPL-2.0 WITH AdditionRef-Progmasoft-Exception-1.0

#include <iostream>
#include <string_view>

int
main()
{
    constexpr std::string_view language = "C++20";
    std::cout << "Hello from " << language << "!\n";
}
